runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -98,6 +100,11 @@ local LanguageTree = {}
 
 LanguageTree.__index = LanguageTree
 
+--- @private
+function LanguageTree:_buf()
```

Comment:
Can we inline this? I think it over-abstracts a bit too much and is only used in one function.